src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,8 @@
 package core.basesyntax;
 
 public class ColorSupplier {
```

Comment:
add here 
 `private final Random random = new Random();`

Reply:
Create private final Random random = new Random(); in ColorSupplier class